src/app/utils/pluralize.ts

```diff
@@ -0,0 +1,6 @@
+export function countAndNoun(count: number, noun: string) {
+    const pluralize = (word: string, count: number): string => {
```

Comment:
As `countAndNoun` are currently placed in a separate module, it might be better to make `pluralize` as a private function (not exported).

Reply:
However, the current version works as expected, so it's not a necessary change.